app/assets/stylesheets/custom.css.scss

```diff
@@ -895,6 +895,18 @@ $grayMediumLight: #eaeaea;
   /*float: right;*/
 }
 
+.rounded-box {
+  position: relative;
+  padding: 10px 10px 10px;
```

Comment:
Shorthand form for property `padding` should be written more concisely as `10px` instead of `10px 10px 10px`